server/controllers/methods.js

```diff
@@ -0,0 +1,9 @@
+const db = require('../dbconnection/schema.js'); //accese to your scheema;
+
+exports.addData = (req,res) => {
+	db.create(req.body).then(results => res.send(results))
+};
+exports.getData = (req,res) => {
+	db.findOne({_id: req.params.id})
+		.then(singleData => res.send(singleData));
+};
```

Comment:
This is a neat way to keep your server file clean of response functions!  I like it!

Reply:
thanks!